data/web/autoconfig.php

```diff
@@ -39,51 +39,20 @@
          <username>%EMAILADDRESS%</username>
          <authentication>password-cleartext</authentication>
       </incomingServer>
-      <incomingServer type="imap">
-         <hostname><?=$autodiscover_config['imap']['server']; ?></hostname>
-         <port><?=$autodiscover_config['imap']['tlsport']; ?></port>
-         <socketType>STARTTLS</socketType>
-         <username>%EMAILADDRESS%</username>
-         <authentication>password-cleartext</authentication>
-      </incomingServer>
-
-<?php
-$records = dns_get_record('_pop3s._tcp.' . $domain, DNS_SRV); // check if POP3 is announced as "not provided" via SRV record
```

Comment:
I think being able to hide POP3 was useful. In fact, I was the person who added this code seven years ago. However, I do understand that you want to deal with situations where the DNS might unreliable. So instead of unconditionally presenting both POP3S and IMAPS, perhaps we could either have a configuration switch somewhere, or, perhaps more conveniently, simply query our database and hide the protocol if it is disabled for all mailboxes (and all their app passwords) of the domain.